### PR TITLE
Running scenarii and applying dynamic strategies

### PR DESCRIPTION
**Pull request by thom4parisot**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,14 +6,18 @@
...
-      
+
     <title> See you later elevator </title>
   </head>
 
   <body>
       <div class="building column">
         <dl class="shaft">
-          <dt class="elevator" data-at-floor="1">Elevator</dt>
+	        <dt class="elevator" data-at-floor="1">elevators[0]</dt>
```
Comment (DavidBruant) [inline index.html]: Vous gagnez 317 points Internet pour cette idée excellente.

File: src/main.css [hunk 2]
```diff
@@ -4,6 +4,10 @@
   width: 50%;
 }
 
+.hidden{
```
Comment (DavidBruant) [inline src/main.css]: bikeshed, but [hidden attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#hidden)?

File: src/onload.js [hunk 3]
```diff
@@ -14,20 +14,96 @@
...
+  function prepare(levelElement){
+    var scenario = scenarii[levelElement.value - 1];
+
+    if (!scenario){
+      return;
+    }
+
+    [].slice.call(d.querySelectorAll('[data-level]')).forEach(function(floor){
+      if (floor.getAttribute('data-level') > scenario.floors){
+        floor.classList.add('hidden');
```
Comment (DavidBruant) [inline src/onload.js]: hidden attr followup here (.setAttribute('hidden', true))

**Comment by DavidBruant** [comment]
Excellent PR just from the code :-)
Testing now.

File: src/onload.js [hunk 4]
```diff
@@ -14,20 +14,96 @@
...
+      }
+    });
+
+    [].slice.call(d.querySelectorAll('.shaft')).forEach(function(shaft, i){
+      if (parseInt(i, 10) >= scenario.elevators){
+        shaft.classList.add('hidden');
+      }
+      else{
+        shaft.classList.remove('hidden');
+      }
```
Comment (DavidBruant) [inline src/onload.js:80]: Maybe need to put the elevator at ground floor here?

**Comment by thom4parisot** [comment]
Ready to merge. Implements everything :-)

**Comment by thom4parisot** [comment]
Avec accord de @glovesmore (qui n'a bu que de l'eau).

**Pull request merged**